torchtext/models/roberta/bundler.py

```diff
@@ -192,3 +192,61 @@ def encoderConf(self) -> RobertaEncoderConf:
     Please refer to :func:`torchtext.models.RobertaModelBundle` for the usage.
     '''
 )
+
+
+ROBERTA_BASE_ENCODER = RobertaModelBundle(
+    _path=urljoin(_TEXT_BUCKET, "roberta.base.encoder.pt"),
+    _encoder_conf=RobertaEncoderConf(vocab_size=50265),
+    transform=lambda: T.Sequential(
+        T.GPT2BPETokenizer(
+            encoder_json_path=urljoin(_TEXT_BUCKET, "gpt2_bpe_encoder.json"),
+            vocab_bpe_path=urljoin(_TEXT_BUCKET, "gpt2_bpe_vocab.bpe"),
+        ),
+        T.VocabTransform(
+            load_state_dict_from_url(urljoin(_TEXT_BUCKET, "roberta.vocab.pt"))
+        ),
+        T.Truncate(510),
+        T.AddToken(token=0, begin=True),
+        T.AddToken(token=2, begin=False),
+    ),
+)
+
+ROBERTA_BASE_ENCODER.__doc__ = (
+    '''
+    Roberta Encoder with Base configuration
+
+    Please refer to :func:`torchtext.models.RobertaModelBundle` for the usage.
+    '''
+)
+
+
+ROBERTA_LARGE_ENCODER = RobertaModelBundle(
+    _path=urljoin(_TEXT_BUCKET, "roberta.large.encoder.pt"),
+    _encoder_conf=RobertaEncoderConf(
+        vocab_size=50265,
+        embedding_dim=1024,
+        ffn_dimension=4096,
+        num_attention_heads=16,
+        num_encoder_layers=24,
+    ),
+    transform=lambda: T.Sequential(
+        T.GPT2BPETokenizer(
+            encoder_json_path=urljoin(_TEXT_BUCKET, "gpt2_bpe_encoder.json"),
+            vocab_bpe_path=urljoin(_TEXT_BUCKET, "gpt2_bpe_vocab.bpe"),
+        ),
+        T.VocabTransform(
+            load_state_dict_from_url(urljoin(_TEXT_BUCKET, "roberta.vocab.pt"))
+        ),
+        T.Truncate(510),
+        T.AddToken(token=0, begin=True),
+        T.AddToken(token=2, begin=False),
+    ),
+)
+
+ROBERTA_LARGE_ENCODER.__doc__ = (
+    '''
+    Roberta Encoder with Large configuration
```

Comment:
I recommend to mention what dataset was used to train this model and copy-right information.

https://pytorch.org/audio/stable/pipelines.html#torchaudio.pipelines.WAV2VEC2_ASR_BASE_10M

Reply:
cc: @kartikayk

Reply:
@mthrok just added the model description from fairseq repo. I also put adding citations to torchtext docs as a followup item. Lmk if this looks okay

Reply:
I think it's worth mentioning the names of the datasets. Reading the README and the paper, it seems that they used `BOOKCORPUS`, `CC-NEWS`, `OPENWEBTEXT` and `STORIES` for pre-training. Can you double-check?

Reply:
> I also put adding citations to torchtext docs as a followup item.

Sounds good. use of `sphinx.bibtex` extension makes this easy.

https://github.com/pytorch/audio/blob/main/docs/source/refs.bib

Reply:
> I think it's worth mentioning the names of the datasets. Reading the README and the paper, it seems that they used `BOOKCORPUS`, `CC-NEWS`, `OPENWEBTEXT` and `STORIES` for pre-training. Can you double-check?

I just checked again and you are correct, those are the datasets that Roberta is trained on. I just added a paragraph on this in the documentation.

Reply:
LGTM.